### PR TITLE
lz4: add page

### PR DESCRIPTION
**Pull request by janwh**
Hi! As lz4 is IMHO a very nice alternative to gzip or xz compression, it should have a tldr page as well. I hope my suggested examples are good enough even for novice users.  Cheers! 🤙

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/lz4.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+`lz4 -d {{file.lz4}}`
+
+- Decompress a file and write to stdout:
+
+`lz4 -dc {{file.lz4}}`
+
+- Package and compress a directory and its contents:
+
+`tar cvf - {{/path/to/dir}} | lz4 - {{dir.tar.lz4}}`
```
Comment (pxgamer) [inline pages/common/lz4.md]: Could you change the absolute path to `{{path/to/dir}}`?

Comment (janwh) [reply pages/common/lz4.md]: Done!

File: pages/common/lz4.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- Package and compress a directory and its contents:
+
+`tar cvf - {{path/to/dir}} | lz4 - {{dir.tar.lz4}}`
+
+- Decompress and unpack a directory and its contents:
+
+`lz4 -d {{dir.tar.lz4}} | tar -xv`
+
+- Compress a file using the best compression:
```
Comment (agnivade) [inline pages/common/lz4.md]: This should have been the second example.

Comment (sbrl) [reply pages/common/lz4.md]: Ah, oops! That might have been a good idea.

Most of the time, the best compression mode doesn't actually compress it much more than the standard compression does, whilst taking much longer. So to that end it's not actually that useful in day-to-day circumstances.

Comment (janwh) [reply pages/common/lz4.md]: I thought so, too at first. I decided against it as for me LZ4's major benefit is its default mode (`-1`): the space savings on compressable files are nicely balanced with the speed of processing on incompressable files. The improved compression of using `-9` are diminishing the benefits of processing speed, and as tldr is making an effort to appeal to novice users, I thought I'd intentionally discourage the use of `-9` by putting it at the end.

Comment (agnivade) [reply pages/common/lz4.md]: Ok, fair enough.

**Pull request merged**